### DISCUSSION TITLE
Allow Negative Z Values for Sprites in the Pipelined Renderer?

### DISCUSSION THREAD

**Discussion by zicklag** [bevyengine/bevy#2749] [Ideas]
Does it make sense to anybody else to support negative depths for sprites by default? Right now we have a near plane of 0, but I feel like a near plane of -1000 would be useful, so that you have like a "base" depth ( 0 ), and you can put things behind _or_ on top of the base depth. It feels strange to me whenever I make a sprite with a non-positive Z and it disappears.

**Comment by alice-i-cecile** [comment]
I'd prefer a proper layering solution for 2D (see #1275), but I'm in favor of making this change now.
